### PR TITLE
Assignment3

### PR DESCRIPTION
**Pull request by idanwe**

### CODE REVIEW

File: lib/index.js [hunk 1]
```diff
@@ -1,28 +1,29 @@
...
 var Hapi = require('hapi');
-var Hoek = require('hoek');
 var Version = require('./version');
 
 
 // Declare internals
 
-var internals = {};
+var internals = {
+    port: 8000
```
Comment (hueniverse) [inline lib/index.js]: Who's using this?

Comment (idanwe) [inline lib/index.js]: Forgot to remove it

**Comment by idanwe** [comment]
@hueniverse Fixed and rebase the master

File: test/index.js [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+        Server.init(5000, function (err, server) {
+
+            expect(err).to.not.exist();
+            expect(server.info.port).to.equal(5000);
+
+            server.stop(done);
+        });
+    });
+
+    it('handles register plugin errors', function (done) {
```
Comment (EvanMarkHopkins) [inline test/index.js]: #79 
> When monkey patching code in a test, mark that test with the { parallel: false } lab option to make it both safe for future parallel testing as well as visual cue.

Comment (idanwe) [inline test/index.js]: Add it, thanks

**Comment by idanwe** [comment]
Thanks :smile:

**Pull request merged**